Tiltfile

```diff
@@ -230,6 +231,27 @@ def deploy_worker_templates(flavor, substitutions):
 
     yaml = str(read_file(yaml_file))
 
+    # azure account and ssh replacements
+    for substitution in substitutions:
+        value = substitutions[substitution]
+        yaml = yaml.replace("${" + substitution + "}", value)
+
+    # if metadata defined for worker-templates in tilt_settings
+    if "worker-templates" in settings:
```

Comment:
the order here matters because otherwise `substitutions = settings.get("worker-templates").get("flavors").get(flavor, {})` overrides what was previously set from defaults with `substitutions[k] = defaults[k]` without ever replacing the defaults in the yaml template.